src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -265,16 +268,32 @@ export abstract class BaseEslintEngine extends AbstractRuleEngine {
 	/**
 	 * Uses a list of rules to generate an object suitable for use as the "rules" property of an ESLint configuration.
 	 * @param {Rule[]} rules - A list of rules that we want to run
-	 * @returns {[key: string]: ESRuleConfig} A mapping from rule names to the configuration at which they should run.
+	 * @returns {ESRuleConfig} A mapping from rule names to the configuration at which they should run.
 	 * @private
 	 */
-	private configureRules(rules: Rule[]): {[key: string]: ESRuleConfig} {
-		const configuredRules: LooseObject = {};
+	private configureRules(rules: Rule[]): ESRuleConfig {
+		const configuredRules: ESRuleConfig = {rules: {}};
 		rules.forEach(rule => {
 			// If the rule has a default configuration associated with it, we use it. Otherwise, we default to "error".
-			configuredRules[rule.name] = rule.defaultConfig || 'error';
+			configuredRules.rules[rule.name] = rule.defaultConfig || 'error';
 		});
 		return configuredRules;
 	}
 
+	private static validateEnv(env: unknown): env is EslintEnv {
+		if (env == null) {
```

Comment:
Is the idiomatic ts check in that case `!env`?

Reply:
No. `!env` would return `true` if `env` is false, 0, or a few other things. `env == null` only returns true if it's `null` or `undefined`.